tests/acceptance/lazy-router-test.js

```diff
@@ -0,0 +1,20 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'dummy/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | lazy-router');
+
+test('Query params are not stored in cache for bundles that are not yet loaded', function(assert) {
+  visit('/');
+  andThen(()=>
+    visit('package1'));
```

Comment:
The packages were loaded at the second visit of the route at that time. That might be due to some other dependency. Second transition is not needed.